common/common.scss

```diff
@@ -6,18 +6,6 @@
     "Noto Color Emoji";
 }
 
-html.anon {
-  body:not(.discover-home):not(.static-login):not(.has-sidebar-page) {
-    // hide other page content from anon users
-    display: none;
-  }
-  body[class=""] {
-    // the admin login route has no class
-    // so it's a special case
-    display: block !important;
-  }
-}
-
```

Comment:
This is sort of related. If I disable the sidebar via site settings, all pages go blank since this hides the full body. 

With the JS redirect, I don't think we need this anymore. And I'd rather not add another exception here for the topic route.